src/components/reference/BuildFileSymbol.jsx

```diff
@@ -0,0 +1,12 @@
+import CodeBlock from "@theme/CodeBlock";
+export default function BuildFileSymbol({ name, signature, children }) {
+  return (
+    <div className="margin-bottom--lg">
+      <CodeBlock language="python">
+        {/* if there's no signature, the BUILD file symbol is just a constant... but we don't currently have any info about what that constant is */}
```

Comment:
Ah yeah that's a bummer. FWIW we delayed publishing these docs solely because we threw the baby out with the bathwater on documenting constants. 

Thank you for doing this, and for not allowing constants to halt a great improvement 👍👍

Reply:
Ah, do you have a reference to any of that? Just want to make sure I'm across any extra issues that were identified to avoid having to rediscover everything.

Reply:
I think this is what I'm thinking of: https://github.com/pantsbuild/pants/discussions/18117

Reply:
Ah cool, thanks. That seems to have some good thinking, particularly relevant to https://github.com/pantsbuild/pants/issues/20978, but none of it seems like major missing pieces that should block this... so I'll get it in.